docker-compose.yml

```diff
@@ -1245,6 +1246,38 @@ services:
         /arrow/ci/scripts/python_build.sh /arrow /build &&
         /arrow/ci/scripts/integration_turbodbc.sh /turbodbc /build"]
 
+  conda-python-substrait:
+    # Possible $SUBSTRAIT parameters:
+    #  - `latest`: latest release
+    #  - `master`: git master branch, use `docker-compose run --no-cache`
+    #  - `<version>`: specific version available on conda-forge
+    # Usage:
+    #   docker-compose build conda
+    #   docker-compose build conda-cpp
+    #   docker-compose build conda-python
+    #   docker-compose build conda-python-substrait
+    #   docker-compose run --rm conda-python-substrait
+    image: ${REPO}:${ARCH}-conda-python-substrait
```

Comment:
We might want to add ${PYTHON} to the image name if we ever want to use different python versions.

Reply:
I added a Note in the yml file.